lib/artist.rb

```diff
@@ -0,0 +1,33 @@
+class Artist
+  extend Concerns::Findable
+  extend Concerns::ClassMethods
+  include Concerns::InstanceMethods
+  attr_accessor :name, :songs
+  @@all = []
+
+  def initialize(name)
+    @name = name
+    @songs = []
+    save
```

Comment:
Should all new instances be saved?

Reply:
@garettarrowood  Ahh that was my big mistake with the lab! No, since there are tests that try to add a duplicate `Artist`, `Song`, `Genre` which would make duplicates. The only time we want to `save` is when it is a _valid_ new song, artist, genre 🤯